crates/pop-parachains/src/errors.rs

```diff
@@ -9,18 +9,33 @@ pub enum Error {
 	Aborted,
 	#[error("Anyhow error: {0}")]
 	AnyhowError(#[from] anyhow::Error),
+	/// An error occurred while encoding the call data.
```

Comment:
Would be nice if we can add docs for the Error type and all other fields

Reply:
As mentioned above, added comments for the Error type and all the errors introduced in this PR in this commit https://github.com/r0gue-io/pop-cli/commit/7d7df02deee57a437203a1de8f2dbfe52b38b4d7.

For existing methods, I feel this should be addressed in a separate PR, as this one is already a big one and adding unrelated changes would make it more complex to review.